tests/models/llama/test_tokenization_llama.py

```diff
@@ -839,13 +839,13 @@ def test_special_tokens_strip(self):
         self.assertEqual(tokens, ["▁No", "<s>", "▁He"])  # spaces are eaten by rstrip / lstrip
 
 
-@require_tiktoken
-@require_read_token
 class TikTokenIntegrationTests(unittest.TestCase):
     """
     A class that regroups important test to make sure that we properly handle the special tokens.
     """
 
+    @require_tiktoken
+    @require_read_token
```

Comment:
`@require_read_token` behaves slightly different from other `@require_xxx`: it doesn't add a `@unittest.skipUnless`. On my machine these tests were not being run, and I can't find them on CI either (but perhaps I'm not looking in the right place)